src/Jobs/TileSingleOverlay.php

```diff
@@ -55,4 +56,23 @@ public function handle()
             File::deleteDirectory($this->tempPath);
         }
     }
+
+     /**
+     * Generate tiles for the object and put them to temporary storage.
+     *
+     * @param File $file
+     * @param string $path Path to the cached image file.
+     */
+    public function generateTiles($file, $path)
+    {
+        $vipsImage = $this->getVipsImage($path);
+        $sourceSpace = Vips\FFI::vips()->vips_image_guess_interpretation($vipsImage);
```

Comment:
@mzur, I cannot figure out the right import statement for using vips-functions (such as the one in the example). The current setup returns following exception: `Attempt to call undefined C function 'vips_image_guess_interpretation'`.

Is it possible that the php-wrapper of libvips does not support all of the packages functions? An if it does, how would I access the functions correctly?

Reply:
`$vipsImage->interpretation` should be what you want.

Reply:
Thank you! I found the [docs](https://libvips.github.io/php-vips/) for php-vips later on :smile: